health/notifications/gotify/README.md

```diff
@@ -0,0 +1,59 @@
+<!--
+title: "Send notifications to Gotify"
+description: "Send alerts to your Gotify instance when an alert gets triggered in Netdata."
+sidebar_label: "Gotify"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notifications/gotify/README.md
+-->
+
+# Send notifications to Gotify
+
+[Gotify](https://gotify.net/) is a self-hosted push notification service created for sending and receiving messages in real time.
+
+## Configuring Gotify
+
+#### Prerquisites
+To use Gotify as your notification service, you need an application token. 
```

Comment:
```suggestion
### Prerequisites

To use Gotify as your notification service, you need an application token. 
```

Reply:
@kickoke I see that going heading level2->level4 wasn't a typo. Why? Don't like odd numbers? 😄

Reply:
I know I know about separation of concerns and all. Logically, it would be an h3 but I wanted it to be an h4 for aesthetics.